src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp

```diff
@@ -155,7 +155,16 @@ tnsr::A<DataType, SpatialDim, Frame> spacetime_normal_vector(
 /*!
  * \ingroup GeneralRelativityGroup
  * \brief  Computes extrinsic curvature from metric and derivatives.
- * \details Uses ADM evolution equation for spatial metric.
+ * \details Uses the ADM evolution equation for the spatial metric,
+ * \f[ K_{ij} = \frac{1}{2N} \left ( -\partial_0 g_{ij}
+ * + N^k \partial_k g_{ij} + g_{ki} \partial_j N^k
+ * + g_{kj} \partial_i N^k \right ) \f]
+ * where \f$K_{ij}\f$ is the extrinsic curvature, \f$N\f$ is the lapse,
+ * \f$N^i\f$ is the shift, and \f$g_{ij}\f$ is the spatial metric. In terms
+ * of the Lie derivative of the spatial metric with respect to a unit timelike
+ * vector \f$n^i\f$ normal to the spatial slice, this corresponds to the sign
+ * convention
+ * \f[ K_{ij} = - \frac{1}{2} \mathcal{L}_{\mathbf{n}} g_{ij} \f]
```

Comment:
I think 4D indices would be better here, to make clear that this is not a spatial Lie derivative.

Reply:
@10220 please reply

Reply:
Sorry, I completely forgot about this! I'll take care of this all now